### PR TITLE
Adding fpfs recipe

### PR DESCRIPTION
**Pull request by mr-superonion**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mr-superonion** [comment]
Hello @conda-forge/help-python 
The recipe has passed all the checks and is ready for your review.
Best regards

**Comment by mr-superonion** [comment]
@conda-forge-admin This is ready for review, thanks!

**Comment by mr-superonion** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/fpfs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "fpfs" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/mr-superonion/FPFS/
```
Comment (ngam) [inline recipes/fpfs/meta.yaml:9]: Let's use tar balls instead if possible pls

File: recipes/fpfs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```
Comment (ngam) [inline recipes/fpfs/meta.yaml:20]:
```suggestion
    - python >=3.7
```

File: recipes/fpfs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/mr-superonion/FPFS/
+  git_tag: 976a2c8b2393a0f872709aee5ad099e455b3a49a
+
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (ngam) [inline recipes/fpfs/meta.yaml:15]:
```suggestion
  noarch: python
```

File: recipes/fpfs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
```
Comment (ngam) [inline recipes/fpfs/meta.yaml:24]:
```suggestion
    - python >=3.7
```

File: recipes/fpfs/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    - numpy
+    - numba
+    - scipy
+    - galsim
+
+test:
+  imports:
+    - fpfs
```
Comment (ngam) [inline recipes/fpfs/meta.yaml:32]:
```suggestion
test:
  imports:
    - fpfs
  requires:
    - pip
  commands:
    - pip check
```

**Review by ngam — CHANGES_REQUESTED** [review]
almost there

**Pull request closed without merging**